feat-content.html

```diff
@@ -31,5 +23,13 @@ <h4 class="feat-content-title">Introduction to Dockstore Webinar</h4>
             <p class="feat-card-description">A simple overview of the Dockstore platform and features. Covers the fundamentals of finding workflows and provides examples of running analysis with sample data.</p>
         </a>
     </div>
+        
+    <div class="feat-content-card">
+        <a class="cardClick" target="_blank" rel="noopener noreferrer"
+           href="https://www.youtube.com/watch?v=VE2J6-4wSNM">
+            <h4 class="feat-content-title">Data Analysis on AnVIL: Use Cases</h4>
+            <p class="feat-card-description">A brief introduction on Data Analyis on AnVIL highlighting some example use cases and with short introduction of tools and tutorials.</p>
```

Comment:
I would have `data analysis` - no caps and `analysis` has a typo

`with short introduction of tools and tutorials.` -- Grammatically, probably `with a short introduction to tools and tutorials`, adding 'a` and maybe changing `of` to `to`.

But more generally, and it may not be worth spending too much time on this, like I'm doing, I feel the video shows how Dockstore fits in the AnVIL ecosystem, so maybe

```
A brief introduction to data analysis on AnVIL using Dockstore and other AnVIL components.
```

(but I don't like how I have AnVIL twice)

Reply:
Thanks for catching the typo. I updated with your suggestions and ran the result by Beth. Ended up with a new line (ran spell checker on it this time).